po/es.po

```diff
@@ -402,7 +402,7 @@ msgstr "Tipos de datos recursivos"
 
 #: src/SUMMARY.md:122 src/std/box-niche.md:1
 msgid "Niche Optimization"
-msgstr "Optimización de nichos"
+msgstr "Optimización del Memória"
```

Comment:
"de memoria" or "de la memoria" - if this is about optimizing a program in the memory dimension, it's better to use "de memoria". If we are optimizing memory itself, then "de la memoria".
Memoria does not have tilde/accent mark - it's "memoria" not "memória".